en/core-api/e-wallet.md

```diff
@@ -1,76 +1,75 @@
 # Core API E-Wallet Integration
 <hr>
 
-?> This payment method is compatible with [QR Code Indonesian Standard (QRIS)](https://www.bi.go.id/id/ruang-media/siaran-pers/Pages/SP_216219.aspx), and can be paid with **any QRIS compatible e-wallet or banking app**.
+Midtrans facilitates its customers by *E-Wallet*. This payment method is compatible with [QR Code Indonesian Standard (QRIS)](https://www.bi.go.id/id/ruang-media/siaran-pers/Pages/SP_216219.aspx). Customers can make payments with GoPay, ShopeePay or any QRIS compatible e-wallet or banking app. 
```

Comment:
> Midtrans facilitates its customers by *E-Wallet*.
No, Midtrans does not. Midtrans just simply allow merchants accept payment from customer via e-wallets that customer have.